### DISCUSSION TITLE
Is it possible to easily make the replacement text dependent on the replaced text?

### DISCUSSION THREAD

**Discussion by mmoody-vv** [microsoft/presidio#1451] [Q&A]
Let's say I have an analyzer that finds lists of four or more numbers. I would like each individual number to be replaced by [redacted].

So:
`Four Five Six Seven` => `[redacted] [redacted] [redacted] [redacted]`
`One Two Three Four Five` => `[redacted] [redacted] [redacted] [redacted] [redacted]`

My first thought is I have to rewrite the Anonymizer engine, but I'm hoping there is a simpler solution I'm just not seeing.

Thanks.

**Comment by omri374** [comment]
Maybe a custom operator can help here?
